hw3/02-charts.js

```diff
@@ -28,25 +28,78 @@ const borderColors = [
 
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const houses = [];
+const numbOfHouseMembers = [];
+
+const parseData = async () => {
+  await fetch(url)
+  .then(response => {
+    console.log('Success', response);
+    return response.json();
+  })
+  .then(data => {
+    data.forEach((character) => {
```

Comment:
why are only House Lannister and House Targaryen hard-coded in? I like your method of filtering. I actually just hard-coded most of it. it is just not clear to me why House Lannister and House Targaryen are exceptions to the filtering.

Reply:
I only hardcoded those because it was the only misspelled houses that I saw and couldn't think of a way to check for it without hardcoding them.